cypress/helpers/util.ts

```diff
@@ -25,7 +25,7 @@ const batchId: string =
     : Cypress.env('CYPRESS_COMMIT') || Date.now().toString());
 
 export const mermaidUrl = (
-  graphStr: string,
+  graphStr: string | string[],
```

Comment:
As this is a type only change, does the viewer support arrays currently?

Reply:
Yes, and it was a pleasant surprise for me